kore/src/Kore/Log.hs

```diff
@@ -95,9 +95,10 @@ withMainLogger reportDirectory koreLogOptions = runContT $ do
             LogStdErr -> pure Colog.logTextStderr
             LogFileText logFile -> ContT $ Colog.withLogTextFile logFile
     let KoreLogOptions{timestampsSwitch} = koreLogOptions
+        KoreLogOptions{logFormat} = koreLogOptions
```

Comment:
Why not just `KoreLogOptions{timestampsSwitch, logFormat} = koreLogOptions`?

Reply:
It's more neat this way. We already use this coding style in other parts of the code.
This keeps lines shorter and it's easier to digest in git diff's as well, since it's a line addition rather than a line change.

Reply:
Ok, makes sense.